### PR TITLE
[stable/nfs-client-provisioner] Add OWNERS file

### PR DESCRIPTION
**Pull request by verwilst**
**What this PR does / why we need it**:
Just adding an OWNERS file..

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Adds OWNERS

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by davidkarlsen** [comment]
@unguiculus @prydonius bump

**Comment by davidkarlsen** [comment]
@lachie83 ping

**Comment by verwilst** [comment]
bump

File: stable/nfs-client-provisioner/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: 2.1.1
+appVersion: 3.0.1
```
Comment (prydonius) [inline stable/nfs-client-provisioner/Chart.yaml:2]: why does the appVersion need to be changed in this PR?

Comment (verwilst) [reply stable/nfs-client-provisioner/Chart.yaml:2]: Because it was wrong in the previous, we're installing 3.0.1 ( image tag )

**Pull request merged**